src/Helpers/RecognizeClassHelper.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Blumilk\BLT\Helpers;
+
+use Illuminate\Support\Str;
+
+class RecognizeClassHelper
+{
+    public static function recognizeObjectClass(string $objectName): string
+    {
+        if (strpos($objectName, "\\")) {
+            return $objectName;
+        }
+
+        $objectName = Str::ucfirst(Str::singular($objectName));
+
+        return self::getObjectNamespace($objectName) . $objectName;
+    }
+
+    public static function getObjectNamespace(string $objectName): string
+    {
+        $type = Str::plural(self::guessType($objectName));
+
+        return "App\\$type\\";
+    }
```

Comment:
Like I said somewhere else, we cannot assume that `App` is main namespace in Laravel applications. Vide https://github.com/blumilksoftware/toby/blob/main/app/Helpers/ColorGenerator.php

This of course could a default value here, but you should add a way to override it. Maybe by additional parameter in method signature?